sklearn/preprocessing/data.py

```diff
@@ -137,7 +137,7 @@ def scale(X, axis=0, with_mean=True, with_std=True, copy=True):
 
     """  # noqa
     X = check_array(X, accept_sparse='csc', copy=copy, ensure_2d=False,
-                    warn_on_dtype=False, estimator='the scale function',
+                    warn_on_dtype=None, estimator='the scale function',
```

Comment:
Sorry I told you to use `warn_on_dtype=None` but we should just not pass `warn_on_dtype` anymore actually, since the default value (`None`) will be used as expected, and we would have to remove this again in future versions.

So basically: everywhere in the code that was passing `warn_on_dtype=False` should just not pass `warn_on_dtype` at all.

Reply:
done